protocol.md

```diff
@@ -399,7 +399,7 @@ The emoji system is still very unknown, but it seems that they are pulled from a
 It seems that at one point it was previously located somewhere else, since the first returns a 301 and the second responds fine.
 
 
-[https://stream.mattkc.com/chat/emotes/]
+https://stream.mattkc.com/chat/emotes/
```

Comment:
What is this link supposed to be? Maybe we can have a table of all the known substitutions or instructions on how to read the data provided by the link?

Reply:
Link is the images to convert text to the emojis

Reply:
I think we can worry about that later. I'm going to make a docs webpage that github pages can host, so then we can add footnotes and cite our sources more cleanly